bucket/barrier.json

```
@@ -3,6 +3,9 @@
     "description": "Open-source KVM software",
     "homepage": "https://github.com/debauchee/barrier",
     "license": "GPL-2.0-only",
+    "depends": [
+        "extras/vcredist2019"
+    ],
```

```suggestion
    "suggest": {
        "vcredist": "extras/vcredist2019"
    },
```

Suggest is better because many people install vcredist through other means.